app/src/main/java/ch/hikemate/app/ui/guide/GuideScreen.kt

```diff
@@ -0,0 +1,286 @@
+) {
+  Box(modifier = Modifier.fillMaxSize().testTag(GuideScreen.GUIDE_SCREEN)) {
+    LazyColumn(
+        modifier =
+            modifier
+                .fillMaxSize()
+                .padding(padding)
+                .padding(horizontal = GuideScreen.HORIZONTAL_PADDING_DP.dp)) {
+          // The header which says HikeMate Guide with the logo
+          item {
+            Spacer(modifier = Modifier.height(GuideScreen.HEADER_SPACING_DP.dp))
+            GuideHeader()
+          }
+          // The app's related topics
+          items(appTopics) { topic ->
+            ExpandableTopicCard(
+                topic = topic,
+                navigationActions = navigationActions,
+            )
+          }
+
+          // Separator between the app's topics and the hiking's topics
+          item { HikingGuideSection() }
+
+          // Hiking in general related topics
+          items(hikingTopics) { topic ->
+            ExpandableTopicCard(
+                topic = topic,
+                navigationActions = navigationActions,
+            )
+          }
+
+          item { Spacer(modifier = Modifier.height(GuideScreen.HEADER_SPACING_DP.dp)) }
+        }
+  }
+}
+
+@Composable
+private fun GuideHeader() {
+  Row(
+      verticalAlignment = Alignment.CenterVertically,
+      modifier =
+          Modifier.padding(vertical = GuideScreen.CONTENT_PADDING_DP.dp)
+              .testTag(GuideScreen.GUIDE_HEADER)) {
+        // HikeMate's logo
+        Image(
+            painter = painterResource(id = R.drawable.app_icon),
+            contentDescription = stringResource(R.string.guide_app_logo),
+            modifier = Modifier.size(GuideScreen.ICON_SIZE_DP.dp))
+        Spacer(modifier = Modifier.width(GuideScreen.HEADER_SPACING_DP.dp))
+        Text(
+            text = stringResource(R.string.guide_title),
+            style = MaterialTheme.typography.headlineMedium)
+      }
+}
+
+@Composable
+private fun HikingGuideSection() {
+  Spacer(modifier = Modifier.height(GuideScreen.SECTION_SPACING_DP.dp))
+  Text(
+      text = stringResource(R.string.guide_hiking_section_title),
+      style = MaterialTheme.typography.headlineMedium)
+  Spacer(modifier = Modifier.height(GuideScreen.HEADER_SPACING_DP.dp))
+}
+
+/**
+ * The composable function for a single topic card
+ *
+ * @param topic the topic which will be described
+ * @param navigationActions
+ */
+@Composable
+private fun ExpandableTopicCard(
+    topic: GuideTopic,
+    navigationActions: NavigationActions,
+) {
+  var isExpanded by remember { mutableStateOf(false) }
+
+  Card(
+      modifier =
+          Modifier.fillMaxWidth()
+              .padding(vertical = GuideScreen.CARD_SPACING_DP.dp)
+              .animateContentSize()
+              .testTag("${GuideScreen.TOPIC_CARD}_${topic.titleResId}"),
+      shape = RoundedCornerShape(GuideScreen.CORNER_RADIUS_DP.dp)) {
+        TopicCardContent(
+            topic = topic,
+            isExpanded = isExpanded,
```

Comment:
Question: Do you think its realistic, to make all other open cards collapse when one is expanded? This way it can not happen, that the user has all the cards expanded and needs to scroll for a long time to reach a different card. If it takes too much time, this could be a task for a future sprint.

Reply:
From what I've tested this is not possible since when the topic card disappears from the screen it is directly closed. 
Tell me if this is something you found while testing